### DISCUSSION TITLE
Login error in Dashboard

### DISCUSSION THREAD

**Discussion by lxbxinwei** [fastapi/full-stack-fastapi-template#1114] [Questions] [answered]
Hello, @tiangolo 

I follow all steps in [full-stack-fastapi-template (latest)](https://github.com/tiangolo/full-stack-fastapi-template/tree/master)，the code run in a remote machine, and I login from another machine. 

The problem is, I can go to `ip:8090`, `ip:8080`, `ip/docs`, `ip/redoc`. However, I can not go to `ip`, the main page of the project. After I enter email and password, it just circle all the time, did not forward.

Is there anything I did wrong?

Best regards.

<img width="241" alt="Snipaste_2024-03-15_16-29-44" src="https://github.com/tiangolo/full-stack-fastapi-template/assets/138550108/68303ec1-2e8e-496e-b3d3-eba61d10151b">

**Comment by SaurabhXRT** [comment]
Could you please describe the error message you're seeing in the terminal or any other location? Providing details about the error will help in diagnosing and resolving the issue effectively.

**Comment by parthjoshi90** [comment]
I have got the same issue while running it in the local environment.

![Screenshot from 2024-03-16 12-59-32](https://github.com/tiangolo/full-stack-fastapi-template/assets/18608298/9bca5e21-5642-4e3c-af8d-e8edb65402cf)

**Comment by lxbxinwei** [comment]
> Could you please describe the error message you're seeing in the terminal or any other location? Providing details about the error will help in diagnosing and resolving the issue effectively.

I think I know the problem, it seems like network error, but I have no idea to fix it.

When I try login on the server side, via `localhost`, everything works well.

If I login on the remote device via `ip`, it can not work.

and the error in console(remote device) is👇 (look at the red box)
![Pasted image 20240318094806](https://github.com/tiangolo/full-stack-fastapi-template/assets/138550108/d9e8390c-6c86-472c-9390-ee5fbc31b1e1)

**Comment by lxbxinwei** [comment, marked as answer]
I delete the frontend images, and `docker compose -f docker-compose.yml up -d` again, everything works

**Comment by tiangolo** [comment]
Thanks for reporting back!

**Comment by donpolyp** [comment]
sorry, im getting the same issue. And re-building the images didnt help me. Some searching around says maybe this is CORS related? 
Heres a screenshot of the error after I replicated when running the frontend only in dev mode:
<img width="1271" alt="login_error" src="https://github.com/tiangolo/full-stack-fastapi-template/assets/7856187/67ce965b-3c55-4cfd-9853-43c8775e7194">

**Reply by donpolyp** [reply]
gran admirador de tu trabajo @tiangolo

**Reply by lxbxinwei** [reply]
No need to change CORS policy and frontend code. You just check the `DOMAIN` in server's `.env`, which is the same as you add to hosts file in your device.

**Comment by parthjoshi90** [comment]
Hello @donpolyp

can you try adding this to your local `/et/hosts` file?

![image](https://github.com/tiangolo/full-stack-fastapi-template/assets/18608298/10bc4404-71ac-4a4d-8736-ad2fce36d0ff)

**Reply by fn-hide** [reply]
Thanks @parthjoshi90 
You help me to resolve the issue by adding `mydomain.com` into `/etc/hosts` then CORS error will be dissapear.

**Reply by fn-hide** [reply]
Maybe the `deployment.md` should be update too @tiangolo

**Comment by Ryannnnnlye** [comment]
Hi, I am facing the same issue too. The code is running in an Ubuntu server, and I am using a Windows machine to remote SSH through VSC to make code changes. I can access the login pages, sign up pages, :8090, :8080 etc using the ip address. However, when i try to sign up for an account, i receive this error:
![Screenshot 2024-11-04 at 10 26 41 PM](https://github.com/user-attachments/assets/2e7d6563-111b-45ea-8d2f-542aaaab30e7)

I've only changed the DOMAIN variable in frontend > .env file, and did not touch the other "localhost" specified in the .env file. Environment is still set to local. Additionally, I can't find the router details in the screenshot attached by @parthjoshi90 .
